source/renderer/app/types/sidebarTypes.js

```diff
@@ -12,3 +12,24 @@ export type SidebarWalletType = {
   isNotResponding: boolean,
   hasNotification: boolean,
 };
+
+export type WalletSortByOptions = 'DATE' | 'BALANCE' | 'NAME' | 'NONE';
+
+export type WalletSortOrderOptions = 'asc' | 'desc';
+
+export const WalletSortBy: EnumMap<string, WalletSortByOptions> = Object.freeze(
```

Comment:
I think we might not need `Object.freeze` as long as we have typing system (flow) and common sense (trust in developers).

`Object.freeze` will work in this case for now but it only gives us a shallow immutability.

In TypeScript there are additional 3rd party helpers that allow `DeepReadonly` which makes it safe without extra code in the bundle.

Let me know your thoughts.

Reply:
I guess we can remove it.